### DISCUSSION TITLE
Projeto de API integrada a python

### DISCUSSION THREAD

**Discussion by ThiagoTemoteo** [community/community#123538] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

“Sou estudante de um curso técnico e estou precisando de ajuda na matéria de programação em Python. Meu professor me passou um projeto no qual devo usar uma API de alguma rede social e integrá-la ao Python. A ideia do meu grupo é capturar dados como o número de compartilhamentos, curtidas, reposts e talvez mais algumas informações. Pensei no Instagram e fui atrás para ver se conseguia encontrar alguma solução. No entanto, não encontrei nenhum vídeo na internet e o guia oficial da API no site do Meta Developer está sendo muito confuso para mim. Alguém sabe me dizer se esse projeto é viável e se da para levar para frente ?

Obs: tenho 3 semanas apartir de hoje.

**Comment by grandmas-favorite** [comment]
Hi @thibasBigT, thanks for participating!

We take our [[Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
